package.json

```diff
@@ -5,9 +5,9 @@
   "main": "dist/index.js",
   "scripts": {
     "prestart": "npm install",
-    "start": "webpack-dev-server --progress",
+    "start": "node_modules/.bin/webpack-dev-server --progress",
```

Comment:
This is not needed, Webpack uses the local package if it exists any way, especially so we could write it more cleanly